apps/pie-docs/src/assets/styles/components/_homePage.scss

```diff
@@ -0,0 +1,110 @@
+@use '../fozzie-config' as f;
+
+.c-hero {
```

Comment:
<img width="1082" alt="Screenshot 2023-07-26 at 08 38 49" src="https://github.com/justeattakeaway/pie/assets/16766185/92fd4097-876f-462a-bd90-7d40e1f59701">

I think the name hero is being slightly misused here. The hero is generally a banner at the top of the website, not the entire home page container, which this appears to be

Reply:
yeah I wasn't sure what to call it 🤔 I could revert it back to `c-homePage` if that makes more sense?

Reply:
I've refactored the classnames now - I think it makes it clearer 👍 🤞